cmd/azure-config-credentials-injector/credentials_injector_test.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func Test_mergeCloudConfig(t *testing.T) {
+
+	tmpDir, err := ioutil.TempDir("", "cccmo-azure-creds-injector")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
You could use `require.NoError(t, err)`, require is the same as assert except it calls fatal under the hood instead of error.